src/AutoFixture.TUnit/ArgumentsAutoDataAttribute.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using AutoFixture.TUnit.Internal;
+
+namespace AutoFixture.TUnit;
+
+/// <summary>
+/// Provides a data source for a data theory, with the data coming from inline
+/// values combined with auto-generated data specimens generated by AutoFixture.
+/// </summary>
+[SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes",
+    Justification = "This attribute is the root of a potential attribute hierarchy.")]
+public class ArgumentsAutoDataAttribute : AutoFixtureDataSourceAttribute
```

Comment:
In the newer versions of AF I'm trying to keep the semantics of the integrated library. `AutoDataAttribute` comes from xUnit's `DataAttribute`. Since you call it "arguments" in TUnit, I'm thinking to rename it to `AutoArgumentsAttribute`. What do you think?

Reply:
Yeah that works for me. I guess that'd be just for TUnit? Makes sense to keep the naming close to the framework

Reply:
Yes, these attributes will be present only in the TUnit integration.